compose/cli/command.py

```diff
@@ -80,6 +80,7 @@ def get_client(environment, verbose=False, version=None, tls_config=None, host=N
         version=version, tls_config=tls_config, host=host,
         environment=environment, tls_version=get_tls_version(environment)
     )
+    warn_for_swarm_mode(client)
```

Comment:
OK, that makes sense to me.

> If someone has gone ahead and run up and knows what to expect, we can let them proceed without warning them ever time.

We could check for existing containers, and only print the warning if there aren't any?